tests-e2e/ocw-ci-test-course/nav-links.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { test, expect } from "@playwright/test"
+import { CoursePage } from "../util"
+
+test("Navbar links working properly", async ({ page }) => {
+  const course = new CoursePage(page, "course")
+  await course.goto("")
+
+  const navbar = await page
+    .locator("#course-main-content div")
+    .filter({
+      hasText:
+        "Section 1 Menu Title Subsection 1a Menu Title Subsection 1b Menu Title Section 2",
+    })
+    .nth(3)
+  const linkCount = await navbar.locator("a").count()
+  for (let i = 0; i < linkCount; i++) {
+    const link = await navbar.locator("a").nth(i)
+    const url = await link.getAttribute("href")
+    const goto_url = url?.replace("/courses/ocw-ci-test-course/", "")
+    const response = await course.goto(goto_url)
+    await expect(response?.status()).toBeLessThan(400)
+    await expect(page.url()).toContain(url)
+  }
+})
+
+test("Navbar Expanding Properly", async ({ page }) => {
```

Comment:
This test fails on my machine.

Reply:
Added  1 sec delay for the next button click in [b815634](https://github.com/Anas12091101/ocw-hugo-themes/pull/3/commits/b815634930cdaae13351561260fc6eabf5b5a50e) without this test runs so fast and the next click event occurs when the button is disabled and the test was failing